hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/states/endpoint/HeartbeatEndpointTask.java

```diff
@@ -452,7 +452,7 @@ private void processReregisterCommand() {
         LOG.debug("Received SCM notification to register."
             + " Interrupt HEARTBEAT and transit to REGISTER state.");
```

Comment:
Can we update the log to the GETVERSION state?
```suggestion
            + " Interrupt HEARTBEAT and transit to GETVERSION state.");
```

Reply:
Done.